resources/pre-upgrade.sh

```diff
@@ -40,9 +41,61 @@ function movePluginsToTempDir() {
 }
 
 # versionXLessOrEqualThanY returns true if X is less than or equal to Y; otherwise false
-# This code origins from https://stackoverflow.com/a/4024263/12529534
 function versionXLessOrEqualThanY() {
```

Comment:
Nice Refactoring!

Reply:
Oh, this comment is no longer valid :grimacing: The URL references the original solution with `sort -v`.
I am going to delete the comment.

Reply:
- [x] done